packages/pyright-internal/src/tests/hoverProvider.test.ts

```diff
@@ -450,30 +450,30 @@ test('hover on operators', async () => {
 //// A() [|/*marker1*/+|] B()
 //// a = 1 [|/*marker2*/+|] 2
 //// a [|/*marker3*/+=|] 3
-//// b = ([|/*marker4*/~|]a [|/*marker5*/&|] 255,)
-//// c = 4 [|/*marker6*/*|] b
-//// d = 5 [|/*marker7*/not|] in c
-//// e = a [|/*marker8*/<|] b[[|/*marker9*/0|]]
-//// f = e [|/*marker10*/or|] [|/*marker11*/not|] a
+//// b = ([|/*marker4a*/~|]a [|/*marker4b*/&|] 255,)
+//// c = 4 [|/*marker5*/*|] b
+//// d = 5 not [|/*marker6*/in|] c
+//// e = a [|/*marker7a*/<|] b[[|/*marker7b*/0|]]
+//// f = e [|/*marker8a*/or|] [|/*marker8b*/not|] a
 //// g = [1, 2, 3]
-//// g[[|/*marker12*/0|]] = h = g[[|/*marker13*/2|]] = h = g[[|/*marker14*/1|]]
+//// g[[|/*marker9a*/0|]] = h = g[[|/*marker9b*/2|]] = h = g[[|/*marker9c*/1|]]
```

Comment:
i noticed when playing around with this example that hover & go to definition work on the second `=` but none of the others. is that correct?

<img width="419" height="187" alt="image" src="https://github.com/user-attachments/assets/adc59689-3282-4a88-b9fd-70b55b374ee9" />

(it also seems to highlight the rest of the assignment to the right which isn't what i was hovering over)

i don't think i fully understand how chained assignments work but the equals here resolving to a `__setitem__` doesn't seem right to me because it's assigning to `h` right? if so, it wouldn't be calling a `__setitem__` here